apps/cvicko-map/src/components/App.tsx

```diff
@@ -61,6 +61,17 @@ export const App = () => {
   const [selectedFeature, setSelectedFeature] = useState<Feature<Point> | null>(null);
   const [isMobile, setMobile] = useState<boolean | null>(null);
 
+  // set event listeners
+  useEffect(() => {
+    const cancelAnimation = (event: KeyboardEvent) => {
+      if (event.keyCode === 27) stopAnimation();
```

Comment:
`keyCode` si deprecated.  Use `event.code === "Escape"` which is also more readable .